extension/src/background/helpers/dataStorage.ts

```diff
@@ -16,6 +16,9 @@ export const dataStorage = {
   setItem: async (setItemParams: SetItemParams) => {
     await browser.storage.local.set(setItemParams);
   },
+  removeItem: async (key: string) => {
+    await browser.storage.local.remove(key);
```

Comment:
@piyalbasu is this right? tbh I'm not sure where the methods for browser.storage.local could be found

Reply:
This is correct. The docs on this are not easy to find.

Here is the doc for chrome.storage: https://developer.chrome.com/docs/extensions/reference/storage/

And browser.storage is based off of that (with, weirdly, a few small differences): https://github.com/mozilla/webextension-polyfill

Reply:
ah gotchya. I saw those docs and totally whiffed on seeing the `remove` method. See it now though thx!